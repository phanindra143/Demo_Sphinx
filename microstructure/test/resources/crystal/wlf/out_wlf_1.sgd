*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   109.6392
  !k_ij(1,2)=     0.3966
  !k_ij(2,2)=   110.1122
  !k_ij(1,3)=     0.3056
  !k_ij(2,3)=     0.0378
  !k_ij(3,3)=   109.8642
MP,Kxx,1,  109.63918
MP,Kyy,1,  110.11224
MP,Kzz,1,  109.86418
MP,Kxx,2,  110.11224
MP,Kyy,2,  109.63918
MP,Kzz,2,  109.86418
MP,Kxx,3,  110.11224
MP,Kyy,3,  109.63918
MP,Kzz,3,  109.86418
MP,Kxx,4,  109.63918
MP,Kyy,4,  110.11224
MP,Kzz,4,  109.86418
MP,Kxx,5,  109.63918
MP,Kyy,5,  110.11224
MP,Kzz,5,  109.86418
MP,Kxx,6,  110.11224
MP,Kyy,6,  109.63918
MP,Kzz,6,  109.86418
MP,Kxx,7,  110.11224
MP,Kyy,7,  109.63918
MP,Kzz,7,  109.86418
MP,Kxx,8,  109.63918
MP,Kyy,8,  110.11224
MP,Kzz,8,  109.86418
MP,Kxx,9,  109.86418
MP,Kyy,9,  109.63918
MP,Kzz,9,  110.11224
MP,Kxx,10,  109.63918
MP,Kyy,10,  109.86418
MP,Kzz,10,  110.11224
MP,Kxx,11,  109.63918
MP,Kyy,11,  109.86418
MP,Kzz,11,  110.11224
MP,Kxx,12,  109.86418
MP,Kyy,12,  109.63918
MP,Kzz,12,  110.11224
MP,Kxx,13,  109.86418
MP,Kyy,13,  109.63918
MP,Kzz,13,  110.11224
MP,Kxx,14,  109.63918
MP,Kyy,14,  109.86418
MP,Kzz,14,  110.11224
MP,Kxx,15,  109.63918
MP,Kyy,15,  109.86418
MP,Kzz,15,  110.11224
MP,Kxx,16,  109.86418
MP,Kyy,16,  109.63918
MP,Kzz,16,  110.11224
MP,Kxx,17,  110.11224
MP,Kyy,17,  109.86418
MP,Kzz,17,  109.63918
MP,Kxx,18,  109.86418
MP,Kyy,18,  110.11224
MP,Kzz,18,  109.63918
MP,Kxx,19,  109.86418
MP,Kyy,19,  110.11224
MP,Kzz,19,  109.63918
MP,Kxx,20,  110.11224
MP,Kyy,20,  109.86418
MP,Kzz,20,  109.63918
MP,Kxx,21,  110.11224
MP,Kyy,21,  109.86418
MP,Kzz,21,  109.63918
MP,Kxx,22,  109.86418
MP,Kyy,22,  110.11224
MP,Kzz,22,  109.63918
MP,Kxx,23,  109.86418
MP,Kyy,23,  110.11224
MP,Kzz,23,  109.63918
MP,Kxx,24,  110.11224
MP,Kyy,24,  109.86418
MP,Kzz,24,  109.63918
*endif
