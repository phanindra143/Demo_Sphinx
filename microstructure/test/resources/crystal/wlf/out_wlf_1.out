            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\Rein_ohne_Poren\40-60
             3.00000      -1713128.50860         -4562.58652        -12887.95875
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           -15.53346            73.93816
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         -5580.93846      -1720500.57023          1683.06682
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000            -4.89686         15625.00000           -22.41489
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         -9278.24799          4112.21617      -1716639.09202
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000            41.13499           -41.11884         15625.00000
  /%
